index.html

```diff
@@ -5,9 +5,92 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Currency Converter</title>
     <link rel="stylesheet" href="css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Alegreya:ital,wght@0,400..900;1,400..900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Currency Converter!</h1>
+    <h1>Currency Converter App!</h1>
+
+    <div class="forms-container">
+      <!-- form for adding a new exchange rate -->
+      <form class="form" id="add-rate-form">
+        <h2>Add new exchange rate</h2>
+        <label for="base-currency">Base Currency:</label><br />
+        <input type="text" id="base-currency" name="base-currency" required />
+        <br /><br />
```

Comment:
You can use `margin-bottom` instead of `<br />` to have more control over the output.

Reply:
Thank you. Fixed.